app/src/main/java/ch/sdp/vibester/database/DataGetter.kt

```diff
@@ -52,7 +53,7 @@ class DataGetter @Inject constructor() {
     fun updateFieldInt(uid: String, fieldName: String, newVal: Int, method:String) {
         dbUserRef.child(uid).child(fieldName)
             .get().addOnSuccessListener { t ->
-                var finalVal  = checkValue(t, method, newVal)//newVal TO DELETE IF TESTS PASS, OTHERWISE ROLL BACK!
+                val finalVal  = checkValue(t, method, newVal)//newVal TO DELETE IF TESTS PASS, OTHERWISE ROLL BACK!
```

Comment:
what is the purpose of this comment ? We should not keep it no ?

Reply:
It was not me who left it, i am not exaclty sure what it means... I will ask inva group